contrib/rpmbuild/examples/golang/BUILD

```diff
@@ -0,0 +1,12 @@
+golang_rev = '1.9.2'
+
+rpm_package(
+  name='golang',
+  spec='golang.spec',
+  remote_sources=[
```

Comment:
I think that this should be converted to a `remote_rpm_source`. This is just the situation it was written for, it will download the file from a known namespace and cache it permanently. 

In my local tests, this pares 20+ seconds off each invocation of the `./pants rpmbuild contrib/rpmbuild/examples:golang`, which is critical when debugging a spec file (as you well know) 😛 .

Note that this downgrades to the default golang version from the go contrib module - there should someday be a first-class way to associate these in pants.ini.

```
diff --git a/contrib/rpmbuild/examples/BUILD b/contrib/rpmbuild/examples/BUILD
index 85020bd06..3d72d2607 100644
--- a/contrib/rpmbuild/examples/BUILD
+++ b/contrib/rpmbuild/examples/BUILD
@@ -1,12 +1,17 @@
-golang_rev = '1.9.2'
+golang_rev = '1.8.3'
+
+remote_rpm_source(
+  name='go.tar.gz',
+  version=golang_rev,
+)

 rpm_package(
   name='golang',
   spec='golang.spec',
-  remote_sources=[
-    'https://storage.googleapis.com/golang/go{}.linux-amd64.tar.gz'.format(golang_rev),
-  ],
   defines={
     'version': golang_rev,
   },
+  dependencies =[
+    ':go.tar.gz',
+  ]
 )
diff --git a/contrib/rpmbuild/examples/golang.spec b/contrib/rpmbuild/examples/golang.spec
index fa7509215..9bf8edb3e 100644
--- a/contrib/rpmbuild/examples/golang.spec
+++ b/contrib/rpmbuild/examples/golang.spec
@@ -2,6 +2,8 @@
 # confused by prebuilt Go binaries.
 %global debug_package %{nil}

+%define golang_source_pkg go.tar.gz
+
 Summary: Go language
 Name: golang
 Version: %{version}
@@ -15,7 +17,7 @@ BuildRequires: tar
 # Disable RPM's standard automatic dependency detection since the Go binaries are static.
 AutoReqProv: no

-Source: go%{version}.linux-amd64.tar.gz
+Source: %{golang_source_pkg}

 BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)

@@ -33,7 +35,7 @@ and efficient software.
 %install
 rm -rf "%{buildroot}"
 mkdir -p "%{buildroot}/usr/local"
-tar xzvf "${RPM_SOURCE_DIR}/go%{version}.linux-amd64.tar.gz" -C "%{buildroot}/usr/local"
+tar xzvf "${RPM_SOURCE_DIR}"/%{golang_source_pkg} -C "%{buildroot}/usr/local"

 mkdir -p "%{buildroot}/usr/local/bin"
 cd "%{buildroot}/usr/local/bin"
```